sw/cheri/common/flash-utils.hh

```diff
@@ -38,10 +40,12 @@ class SpiFlash {
   void reset() {
     set_cs(true);
     spi->blocking_write(&CmdEnableReset, 1);
+    spi->wait_idle();
```

Comment:
In long term, when CS is managed by SPI host, we probably want to have `set_cs(false)` automatically execute `wait_idle()`